build.gradle

```diff
@@ -111,6 +112,7 @@ dependencies {
 }
 
 jar {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
Just curious, why did we run into a situation with duplicates?

Reply:
this is for compliancy reasons with gradle 7.0 to suppress warnings